### ISSUE TITLE
Put PDF links in the project metadata 

### ISSUE THREAD

**Issue by dblevins** [microprofile/microprofile-site#59]
We should list the current spec on the right side

**Comment by yeekangc** [comment]
Yes, it will be useful to have the specs listed nicely on the website highlighting the latest versions with pointers to the previous ones as appropriate.

Linking to the various Releases for the various specs is an option too e.g. https://github.com/eclipse/microprofile-config/releases.
